apis/pipeline/pipeline.yaml

```diff
@@ -4515,6 +4532,46 @@ components:
                 processId:
                     description: Node pool update process ID.
                     type: string
+        
+        ListNodePoolsResponse:
+            type: object
+            properties:
+                nodepoolList:
+                    type: array
+                    description: Nodepool list
+                    items:
+                        $ref: '#/components/schemas/NodePoolListItem'
```

Comment:
I believe we currently return a plain list of items in list-like requests.

Reply:
Fixed in 2f0b6933c42b9be148fd286d10ef06ea928453ef.